### DISCUSSION TITLE
Non-alphabetic sorting?

### DISCUSSION THREAD

**Discussion by melyso** [marcusolsson/obsidian-projects#676] [Help]
Would is be possible to sort entries in a specified (non-alphabetic) order? I know this may in general not be too useful, but the special case of sorting a "relevance" or "priority" field in the order "high"-"medium"-"low" seems ubiquitous enough to take into consideration. Needless to say, this mostly makes sense for fields that can take values from a set with finite elements, not for general text fields.

**Comment by Acylation** [comment]
Typically we associate a number with the label text manully, like `1-high`, `2-medium`. Though not that elegent, it works and does make sense.

Internal support for the feature can be expected, since we naturally have an index when we preset values in the field configuration. This functionality will be incorporated into the implementation of the single select field.
